### DISCUSSION TITLE
email from github going into gmail spam

### DISCUSSION THREAD

**Discussion by LarryMartell** [community/community#43512] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Starting a few days ago, all email I get from github (commits, PRs, dependbot, etc.) is going into my gmail spam folder. Is anyone else experiencing this?

**Comment by LarryMartell** [comment]
Google is saying these messages are going into spam because "The sender hasn't authenticated this message so Gmail can't verify that it actually came from them."

**Reply by arjenpdevries** [reply]
Same problem here.
